compiler/aarch64/codegen/OMRTreeEvaluator.cpp

```diff
@@ -1928,10 +1929,91 @@ OMR::ARM64::TreeEvaluator::BNDCHKEvaluator(TR::Node *node, TR::CodeGenerator *cg
    return TR::TreeEvaluator::unImpOpEvaluator(node, cg);
    }
 
+static TR::Instruction *compareIntsAndBranchForBNDCHK(TR::ARM64ConditionCode  branchType,
```

Comment:
Can you call this `compareIntsAndBranchForArrayCopyBNDCHK` to make it clear what it is intending to support?